test-conclave-init.sh

```diff
@@ -11,11 +11,11 @@ echo
 
 pushd build
 
-conclaveInitJar=$(find repo/com/r3/conclave/conclave-init/ -name 'conclave-init-*.jar')
+conclaveInitJar=$(find repo/com/r3/conclave/conclave-init/ -name 'conclave-init-*jar' -not -name 'conclave-init-*javadoc.jar' -not -name 'conclave-init-*-sources.jar')
```

Comment:
As we are generating javadocs and sources now, this script needs more filtering, to actually find, for example, conclave-init-1.3.jar

Reply:
If we read in the conclave version from versions.gradle then we can be exact about the file name we want, and thus not need narrow done manually.

Reply:
I prefer not to do that to be honest, as it will create a dependency somehow that we do not need. I.e. I see this script as something that in theory we can run with the repo only, without the version file